app/scss/ui/_components.price.scss

```diff
@@ -0,0 +1,300 @@
+    justify-content: center;
+    display: flex;
+    margin-top: 30px;
+    @media only screen and (max-width: 500px) {
+      margin-top: 25px;
+    }
+  }
+  &__start {
+    font-weight: 500;
+    line-height: 24px;
+    font-size: 20px;
+    text-align: center;
+    color: #A6B1DC;
+    padding-right: 15px;
+    display: flex;
+    align-items: center;
+    @media only screen and (max-width: 500px) {
+      font-size: 16px;
+    }
+  }
+  &__price {
+    display: flex;
+    font-weight: bold;
+    line-height: 40px;
+    font-size: 48px;
+    text-align: center;
+    letter-spacing: -0.2px;
+    color: $brand;
+    @media only screen and (max-width: 500px) {
+      font-size: 40px;
+    }
+  }
+  &__time {
+    display: flex;
+    align-items: flex-end;
+    font-weight: bold;
+    line-height: 20px;
+    font-size: 16px;
+    text-align: center;
+    letter-spacing: -0.05px;
+    color: $brand;
+  }
+  &__check-container {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    max-width: 185px;
+    margin: 50px auto;
+    flex-wrap: wrap;
+    &.mg- {
+      margin: 0px auto;
+      max-width: 170px;
+    }
+    @media only screen and (max-width: 500px) {
+      margin: 15px auto;
+    }
+  }
+  &__properties {
+    justify-content: center;
+    display: flex;
+    align-items: center;
+    margin: 10px 0;
+    @media only screen and (max-width: 500px) {
+      margin: 7px 0;
+    }
+  }
+  &__check-text {
+    line-height: 20px;
+    font-size: 14px;
+    letter-spacing: -0.05px;
+    color: #808080;
+    .blue_text {
+      color: #5162AA;
+    }
+  }
+  &__check-img {
+    width: 15px;
+    margin-right: 8px;
+  }
+  &__title-range {
+    line-height: 20px;
+    font-size: 14px;
+    text-align: center;
+    color:#353F80;
+    &.-lightgrey {
+      color: #C4C4C4;
+    }
+    &.-margin {
+      margin-top: 15px;
+      margin-bottom: 40px;
+      @media only screen and (max-width: 500px) {
+        margin-bottom: 30px;
+      }
+    }
+  }
+  &__range {
+    width: 300px;
+    margin: 20px auto;
+    @media only screen and (max-width: 800px) {
+      width: 250px;
+    }
+  }
+  &__range-container {
+    width: 100%;
+    height: 7px;
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    @media only screen and (max-width: 800px) {
+      max-width: 240px;
+    }
+  }
+  &__slider {
+    &::-webkit-slider-thumb {
+      -webkit-appearance: none;
+      appearance: none;
+      width: 15px;
+      height: 20px;
+      border: 1px solid $brand;
+      border-radius: 5px;
+      background-image: url("/assets/img/pricing_plans/grip.png");
+      cursor: pointer;
+    }
+  }
+  &__range-width {
+    -webkit-appearance: none;
+    width: 100%;
+    height: 7px;
+    background: rgba(0, 0, 0, 0);
+    outline: none;
+    opacity: 1;
+    -webkit-transition: .2s;
+    transition: opacity .2s;
+    border-radius: 5px;
+    position: absolute;
+    &.-grayrange {
```

Comment:
just --gray? (we have __range-width as base selector so we know its range related :))